Add openrouter-models-finder extension

jeejeeguan opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

## Screencast

<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

jeejeeguan commented:
Hi @pernielsentikaer,

I've addressed the review comments in the latest commit:

- [ ] Replace all Chinese comments with English translations
- [ ] Extract TOKENS_PER_MILLION as a documented constant
- [ ] Fix action title capitalization (Copy Model ID, View on OpenRouter)
- [ ] Update README to wrap GitHub username in backticks
- [ ] Add CLAUDE.md to .gitignore

LitoMore commented on extensions/openrouter-models-finder/package.json:
```diff
@@ -0,0 +1,47 @@
+  "icon": "icon.png",
+  "author": "jeejeeguan",
+  "license": "MIT",
+  "version": "1.0.0",
```
See https://developers.raycast.com/information/versioning.

We don't use the `version` field for versioning. Let's remove it.

LitoMore commented on extensions/openrouter-models-finder/tsconfig.json:
This looks like an AI-generated configuration. Can you change them to the Raycast recommended configuration?

LitoMore commented on extensions/openrouter-models-finder/.gitignore:
Looks like an AI-generated `.gitignore` file. Please use the Raycast recommended configuration.

LitoMore commented on extensions/openrouter-models-finder/.eslintrc.js:
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  extends: ["@raycast/eslint-config"],
+  rules: {
+    // 可以在这里添加自定义规则
+  },
+};
```
Please use the Raycast recommended configuration.

LitoMore commented on extensions/openrouter-models-finder/CLAUDE.md:
Unused file. May be we can remove it.

jeejeeguan commented:
Hi @LitoMore , thanks for the feedback! I've updated the configuration to follow Raycast's recommended setup:

- **package.json**: Removed the `version` field 
- **tsconfig.json**: Updated to use Raycast's recommended TypeScript configuration
- **.gitignore**: Replaced with the official Raycast recommended .gitignore
- **ESLint config**: Migrated from .eslintrc.js to eslint.config.js using the new flat config format with @raycast/eslint-config@2.0.4
